Sources/PromotedCore/ClientConfig.swift

```diff
@@ -103,6 +103,16 @@ public final class ClientConfig: NSObject {
   /// turns on signposts in Instruments.
   @objc public var osLogEnabled: Bool = false
 
+  /// Whether to set `log_user_id` on logged events (on
+  /// outer `LogRequest`).
+  @objc public var metricsLoggerIncludesLogUserIDs: Bool = true
+
+  /// Whether to set `session_id` on logged events.
+  @objc public var metricsLoggerIncludesSessionIDs: Bool = true
```

Comment:
JFYI - I plan on eventually moving the default version of this to be done server-side.  We might still provide a ~sessionId to allow clients to scope activity.

Reply:
Gotcha.